src/rpc/dev.rs

```diff
@@ -0,0 +1,105 @@
+use std::{collections::HashMap, sync::Arc};
+
+use ckb_types::core::TransactionView;
+use jsonrpsee::{
+    core::async_trait,
+    proc_macros::rpc,
+    types::{error::CALL_EXECUTION_FAILED_CODE, ErrorObjectOwned},
+};
+
+use ractor::{call_t, ActorRef};
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+use tokio::sync::RwLock;
+
+use crate::{
+    ckb::CkbChainMessage,
+    fiber::{network::DEFAULT_CHAIN_ACTOR_TIMEOUT, serde_utils::U64Hex, types::Hash256},
+};
+
+#[serde_as]
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub(crate) struct SubmitCommitmentTransactionParams {
```

Comment:
we should move AddTlc and RemoveTlc from rpc/channel to here?

Reply:
yes, I plan to do it after this PR.